docs/design.md

```diff
@@ -33,7 +33,7 @@ There is a more detailed explanation in the following paragraphs.
 <sup>1 </sup> **Free space fragmentation** is a problem when there appears to
 be enough free capacity for a new namespace, but there isn't a contiguous
 region big enough to allocate it. The PMEM-CSI driver is only capable of
-allocating continguous memory to a namespace and cannot de-fragment or combine
+allocating continuous memory to a namespace and cannot de-fragment or combine
```

Comment:
I think both "contiguous" and "continuous" are valid:
- https://www.merriam-webster.com/dictionary/contiguous "touching or connected throughout in an unbroken sequence").
- https://www.merriam-webster.com/dictionary/continuous "marked by uninterrupted extension in space, time, or sequence"

But "contiguous memory" has more Google hits. Let's use that. Please submit as a separate PR, it's not related to this PR.

Reply:
Ok, I will revert.

Reply:
Done

Reply:
Is that fix in some PR?

Reply:
No, I missed this in my other PR.